build-managed.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+usage()
+{
+    echo "Usage: $0 [platform] [useservergc]"
+    echo
+    echo "platform can be: FreeBSD, Linux, NetBSD, OSX, Windows."
+    echo "useservergc - Switch used by configure the hosted coreclr instance when executing tests."
+    echo
+}
+
+
+__scriptpath=$(cd "$(dirname "$0")"; pwd -P)
+
+__UnprocessedBuildArgs=
+__TestNugetRuntimeId=-distroRid
+__BuildOS=-os
+__TargetOS=-target-os
+__ServerGC=0
+
+while :; do
+    if [ $# -le 0 ]; then
+        break
+    fi
+
+    lowerI="$(echo $1 | awk '{print tolower($0)}')"
+    case $lowerI in
```

Comment:
Is there a reason this cannot be in dir.props and not in the sh script? I would hope all we would need in the sh script is a placeholder for the OS to pass to msbuild.